### DISCUSSION TITLE
Bing indexer recommendation: The description is too long or too short.

### DISCUSSION THREAD

**Discussion by qureshiaquib** [cotes2020/jekyll-theme-chirpy#1830] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I've got bing recommendation after the indexing that every post which I've done is having "The description is too long or too short." this is of high severity which impacts the search indexing of these post
I see below descriptions:
"Change the description in the <meta description> tag in the page source to be between 25 and 160 characters in length."

I don't know how can I fix this. is this to be fixed in the code?

www.azuredoctor.com is my blog.

### Operations you have already tried

I have not tried anything yet, because it might involve edit the page html file which I don't want. wanted to check with the owner before i implement this.

**Comment by kungfux** [comment, marked as answer]
Chirpy automatically generates the page description, which is limited to the first 400 characters of the post text.

You can override the automatically generated description by specifying it in the front matter as follows:

```markdown
---
title: My post title
description: My post description.
---
```

Hopefully, this will do the trick for you in terms of Bing's warnings.

**Reply by qureshiaquib** [reply]
Thanks kungfux, I'll modify my post and see if bing accepts this.
